internal/future/serving/server.go

```diff
@@ -49,8 +49,8 @@ type Params struct {
 	ServeMux             *http.ServeMux
 	handlersForGrpc      []GrpcHandler
 	handlersForGrpcProxy []GrpcProxyHandler
-	grpcListener         *netlistener.ListenerHolder
-	grpcProxyListener    *netlistener.ListenerHolder
+	GrpcListener         *netlistener.ListenerHolder
```

Comment:
Please don't do this. But someone needs to solve the client story.
This is dangerous because the ListenerHolder is a single use object.